Google.PowerShell/Sql/GcSqlInstanceCmdlets.cs

```diff
@@ -1388,4 +1388,159 @@ private Settings PopulateSetting(Settings newSettings)
             return newSettings;
         }
     }
+
+    /// <summary>
+    /// <para type="synopsis">
+    /// Failover a Cloud SQL Instance.
+    /// </para>
+    /// <para type="description">
+    /// Failover the specified Cloud SQL Instance to its failover replica instance.
+    /// </para>
+    /// <para type="description">
+    /// If a Project is specified, it will failover the specified Instance in that Project. Otherwise, failsover the 
+    /// Instance in the Cloud SDK config project. 
+    /// </para>
+    /// <example>
+    ///   <para>Failover the SQL Instance "test1" in the Project "testing."</para>
+    ///   <para><code>PS C:\> Failover-GcSqlReplica -Project "testing" -Instance "test1"</code></para>
+    ///   <br></br>
+    ///   <para>(If successful, the command returns nothing.)</para>
+    /// </example>
+    /// <example>
+    ///   <para>Failover the SQL Instance "test1" with current settings version 3 in the Project "testing."</para>
+    ///   <para>
+    ///     <code>PS C:\> Failover-GcSqlReplica -Project "testing" -Instance "test1" - SettingsVersion 3</code>
+    ///   </para>
+    ///   <br></br>
+    ///   <para>(If successful, the command returns nothing.)</para>
+    /// </example>
+    /// </summary>
+    [Cmdlet("Failover", "GcSqlInstance")]
+    public class FailoverGcSqlInstanceCmdlet : GcSqlCmdlet
+    {
+        private class ParameterSetNames
+        {
+            public const string ByName = "ByName";
+            public const string ByInstance = "ByInstance";
+        }
+
+        private class ErrorCodes
```

Comment:
This class isn't grouping anything. It is merely appending its name to its member. Consider combining the class name and the variable name.